install.sh

```diff
@@ -289,7 +303,8 @@ else
     HOMEBREW_PREFIX="$HOMEBREW_PREFIX_DEFAULT"
   else
     trap exit SIGINT
-    if [[ $(/usr/bin/sudo -n -l mkdir 2>&1) != *"mkdir"* ]]; then
+    # If allowed to run sudo for mkdir and required to provide sudo password: prompt:
+    if /usr/bin/sudo -l mkdir &>/dev/null && ! /usr/bin/sudo -n -v &>/dev/null; then
```

Comment:
```suggestion
    if ! sudo -n -v &>/dev/null; then
```
We can't use `sudo -l mkdir` until `have_sudo_access` is called, or it could prompt for a password. We could use `sudo -n -l mkdir`, but unfortunately `sudo -n -l mkdir` appears to be buggy, as you've shown me.